rplugin/python3/molten/outputbuffer.py

```diff
@@ -69,7 +70,32 @@ def _get_header_text(self, output: Output) -> str:
         else:
             old = ""
 
-        return f"{old}Out[{execution_count}]: {status}"
+        if output.status == OutputStatus.DONE and not output.old:
+            now = datetime.now()
+            start = output.start_time
+            diff = now - start
+
+            time = ""
+
+            days = diff.days
+            hours = diff.seconds // 3600
+            minutes = diff.seconds // 60
+            seconds = diff.seconds - hours * 3600 - minutes * 60
+            microseconds = diff.microseconds
+
+            # Days
+            if days:
```

Comment:
Could you use [strftime](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior) to format this a little more cleanly? it might end up with "0 d" all the time though, so not too worried about it.

Reply:
I think since you get a `timedelta` instead of a `datetime` object when you do math with times you can't use that method.